Wenab/Wenab.Api/Ynab/YnabClient.cs

```diff
@@ -0,0 +1,5718 @@
+//----------------------
+// <auto-generated>
```

Comment:
why doesn't this file have "generated" or something like that in the extension type?

Reply:
No particular reason 🤷